homeassistant/components/geniushub/sensor.py

```diff
@@ -80,3 +89,47 @@ def device_state_attributes(self):
             last_comms).isoformat()
 
         return {**attrs}
+
+
+class GeniusIssue(Entity):
+    """Representation of a Genius Hub sensor."""
+
+    def __init__(self, client, level):
+        """Initialize the sensor."""
+        self._hub = client.hub
+        self._name = GH_LEVEL_MAPPING[level]
+        self._level = level
+        self._issues = []
+
+    async def async_added_to_hass(self):
+        """Set up a listener when this entity is added to HA."""
+        async_dispatcher_connect(self.hass, DOMAIN, self._refresh)
+
+    @callback
+    def _refresh(self):
+        self.async_schedule_update_ha_state(force_refresh=True)
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return self._name
+
+    @property
+    def should_poll(self) -> bool:
+        """Return False as the geniushub devices should not be polled."""
+        return False
+
+    @property
+    def state(self):
+        """Return the number of issues."""
+        return len(self._issues)
+
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        return {self._level + '_list': self._issues}
```

Comment:
Use new style string formatting instead of string concatenation.

Reply:
Done. Docs PR submitted too.